stabilize.md

```diff
@@ -12,12 +12,12 @@ module SYSTEM-STABILIZER
     configuration
       <stabilize>
         <flapStack> .List </flapStack>
-        <flapState>
+        <flap-state>
           <flap-ward> .Map </flap-ward>  // mapping (address => uint) Address |-> Bool
           <flap-bids> .Map </flap-bids>  // mapping (uint => Bid)     Int     |-> Bid
           <flap-kicks> 0   </flap-kicks>
-          <flap-live>  0   </flap-live>
-        </flapState>
+          <flap-live>  1   </flap-live>
+        </flap-state>
```

Comment:
Looks like we should have state for `beg`, `ttl`, and `tau`: https://github.com/makerdao/dss/blob/25110e12b8323a0d1d8f45b81ebf0dbd4d740818/src/flap.sol#L92

Since they can be `file`d.

Reply:
Also, let's use `true` for `<flap-live>`, not `1`. And treat it like a `Bool` wherever it's used.

Reply:
Separate issue